mlrun/common/schemas/notification.py

```diff
@@ -132,8 +132,14 @@ class SetNotificationRequest(pydantic.v1.BaseModel):
     notifications: list[Notification] = None
 
 
+class NotificationSummary(pydantic.v1.BaseModel):
+    failed: int = 0
+    succeeded: int = 0
+
+
 class NotificationState(pydantic.v1.BaseModel):
     kind: str
     err: Optional[
         str
     ]  # empty error means that the notifications were sent successfully
+    summary: NotificationSummary
```

Comment:
Should it be an optional value?

Reply:
@moranbental no, since this is new feature, problems might appear only when migrating between rc, but in actual released version in should not be optional